polylogue/conversation.py

```
@@ -387,18 +447,84 @@ def process_conversation(
+            run_settings=run_settings,
+            source_mime=source_mime,
+            source_size=source_size,
+            collapse_threshold=collapse_threshold,
+            collapse_thresholds=collapse_thresholds,
+            extra_yaml=extra_yaml,
+            attachments=attachments,
+            citations=citations,
+        )
+    except Exception:
+        # Branch transcripts are auxiliary; DB remains the source of truth.
+        pass
```

Ruff (0.15.9)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

(SIM105)